python/python/glide/async_commands/transaction.py

```diff
@@ -2047,6 +2049,60 @@ def zdiffstore(
             RequestType.ZDiffStore, [destination, str(len(keys))] + keys
         )
 
+    def zinterstore(
+        self: TTransaction,
+        destination: str,
+        keys: Union[List[str], List[Tuple[str, float]]],
+        aggregation_type: Optional[AggregationType] = None,
+    ) -> TTransaction:
+        """
+        Computes the intersection of sorted sets given by the specified `keys` and stores the result in `destination`.
+        If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
+        When in cluster mode,`destination` and all keys in `keys` must map to the same hash slot.
```

Comment:
```suggestion
        If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
 
        When in cluster mode, `destination` and all keys in `keys` must map to the same hash slot.
```